src/features/todos.ts

```diff
@@ -1,9 +1,31 @@
 import { Todo } from '../types/Todo';
 
-export const actions = {};
+type SetTodosAction = {
+  type: 'todos/SET',
+  payload: Todo[]
+};
+
+const setTodos = (todo: Todo[]): SetTodosAction => ({
+  type: 'todos/SET',
+  payload: todo,
+});
+
+export const actions = { setTodos };
+
+type State = Todo[] | null;
+type Action = SetTodosAction;
+
+const todosReducer = (
+  initialState: State = null,
+  action: Action,
+): State => {
+  switch (action.type) {
```

Comment:
Consider using constants for action types instead of hardcoding strings like 'todos/SET'. This will help prevent typos and make it easier to manage action types.